index.js

```diff
@@ -0,0 +1,157 @@
+//cd79c8caed5c47f4ae25ec30f7cca674 is API key
+
+
+//Defaults search term to UK news, page 1
+let pageNumberGlobal = 1;
+let searchResult = "uk";
+upDateUrl(pageNumberGlobal, searchResult);
+updateSideBarUrl(pageNumberGlobal);
+
+//Increases page number by 1 everytime the load more button is pressed
+const loadMoreButton = document.querySelector(".load_more_button");
+loadMoreButton.addEventListener("click", function (event){
+    pageNumberGlobal ++;
+    upDateUrl(pageNumberGlobal, searchResult);
+    updateSideBarUrl(pageNumberGlobal);
+    });
+
+
+//Creates and updates search term, defaults page number to 1
+    let searchInput = document.querySelector(".search_form");
+searchInput.addEventListener("submit", function(event){
+    let searchContent = document.querySelector(".search_box").value;
+    let previousArticles = document.querySelectorAll(".article_box");
+    event.preventDefault();
+    searchResult = searchContent;
+    previousArticles.forEach(node => {
+        node.parentNode.removeChild(node);
+    });
+    upDateUrl(1, searchResult);
+});
+
+
+
+//Updates search URL with search terms and page numbers
+function upDateUrl(number, term) {
+    let defaultUrlFirst = "https://newsapi.org/v2/everything?q=";
+    let defaultUrlSecond = "&language=en&sortBy=publishedAt&apiKey=cd79c8caed5c47f4ae25ec30f7cca674&page="
+    newsUrl = defaultUrlFirst + term + defaultUrlSecond + number;
+    generateNews(newsUrl);
+};
+ 
+
+//Fetches news API
+function generateNews(url) {
+    fetch(url)
+        .then(response => {
+            // console.log("promise has been resolved")
+            return response.json();
+        }) 
+        .then(body => {
+            // pullNewsObject(body);
+            createNewsContent(body);
+        });
+    }
+
+
+// Exploratory function to examin keys in news JSON object.
+
+function pullNewsObject(news) {
+    console.log(news);
+    news.articles.forEach(article => {
+        console.log(article);
+    })
+};
+
+
+//Create .innerHTML of each news article div.
+
+function articleTemplate (article) {
+    let authorVar = "";
```

Comment:
Code would be slightly cleaner if declaration and assignment was combined into single step. That would also allow you to use `const` instead of `let`